go/chat/sender.go

```diff
@@ -26,24 +26,30 @@ type BlockingSender struct {
 	globals.Contextified
 	utils.DebugLabeler
 
-	boxer *Boxer
-	store *attachments.Store
-	getRi func() chat1.RemoteInterface
+	boxer          *Boxer
+	store          *attachments.Store
+	getRi          func() chat1.RemoteInterface
+	prevPagination *chat1.Pagination
```

Comment:
Can we call this like `prevPtrPagination`, otherwise it is just kind of ambiguous.